src/site/content/en/blog/betty-crocker/index.md

```diff
@@ -0,0 +1,352 @@
+---
+title: |
+  Better cooking with the Wake Lock API : A&nbsp;case study with Betty Crocker
+subhead: |
+
+authors:
+  - thomassteiner
+date: 2020-04-01
+hero: hero.jpg
+alt: Betty Crocker Cake Mix—Coffee Cinnamon.
+description: |
+
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - case-study
+  - wake-lock
+  - capabilities
+  - fugu
+draft: true
+---
+## About Betty Crocker
+
+For nearly a century, Betty Crocker has been America's source for modern cooking instruction
+and trusted recipe development.
+Since 1921, when (the fictive character) Betty Crocker began answering questions
+about baking by letter, she has been working to teach people to cook.
+Launched in 1997, [BettyCrocker.com](https://www.bettycrocker.com/)
+continues the legacy of adapting, discovering, and sharing the knowledge that home cooks crave.
+Today, the site receives more than 12&nbsp;million visitors per month.
+
+## The retired iOS and Android apps
+
+Launched to [much fanfare](https://consumergoods.com/betty-crocker-launches-cookbook-app) in 2014,
+Betty Crocker recently took their native apps out of the Apple App Store and the Google Play Store
+after it had been mostly neglected and deprioritized.
+For a long time, the Betty Crocker team have preferred adding features to the mobile site
```

Comment:
For a long time, the Betty Crocker team has preferred adding new features to the mobile site instead of the native apps